source/agora/common/Config.d

```diff
@@ -160,6 +163,16 @@ public struct LoggingConfig
     LogLevel log_level = LogLevel.Error;
 }
 
+/// Configuration for URLs to push a data when an event occurs
+public struct EventHandlerConfig
+{
+    /// URLs to push a data when a block is externalized
+    public immutable string[] block_externalized_handler_address;
```

Comment:
It's not a single address, so the word `address` isn't right. Maybe:

```D
block_externalized_handlers;
block_externalized_handler_addresses;
```

Reply:
That's a good point.